internal/services/service_test.go

```diff
@@ -0,0 +1,144 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package services
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/cartesi/rollups-node/internal/logger"
+)
+
+func setup() {
+	logger.Init("debug", false)
+	buildFakeService()
+}
+
+func TestService(t *testing.T) {
```

Comment:
The tests left a zombie fake-service running on my machine. We need to make sure we are cleaning all resources.